tst/testbugfix/2025-10-02-char0-matrixgroups.tst

```diff
@@ -13,3 +13,19 @@ gap> H:=Group( 1/2*[[-1,1,-1,1],[-1,-1,-1,-1],[1,1,-1,-1],[-1,1,1,-1]],
 gap> h := 1/2*[[-1,-1,1,-1],[1,-1,1,1],[-1,-1,-1,1],[1,-1,-1,-1]];;
 gap> h in H;
 true
+
+#
+# Moreover, if given a matrix outside the original group, it may happen that
```

Comment:
Obviously not a blocker but I think there's a verb missing here

Reply:
I don't see it, could you elaborate?